### ISSUE TITLE
Core functions undefined?

### ISSUE THREAD

**Issue by gbishop** [bmewburn/intelephense-docs#19]
I bet I'm missing something simple. I've installed the server with npm install -g intelephense and I can see it running. It marks references to core functions like file_get_contents (and many others) as undefined. I see the stubs down in the lib/stub directory. 

I have attached a log from vim-lsp. I can see them talking. It looks like intelephense is not finding the stubs. 

Am I missing a configuration step?

[lsp.log](https://github.com/bmewburn/intelephense-docs/files/3969675/lsp.log)

**Comment by gbishop** [comment]
I wasn't passing workspace_config. This appears to work.

```
au User lsp_setup call lsp#register_server({
    \ 'name': 'intelephense',
    \ 'cmd': {server_info->['intelephense', '--stdio']},
    \ 'initialization_options': {"storagePath": "/tmp/intelephense"},
    \ 'whitelist': ['php'],
    \ 'root_uri':{server_info->lsp#utils#path_to_uri(
    \             lsp#utils#find_nearest_parent_file_directory(
    \             lsp#utils#get_buffer_path(),
    \             ['.git/']
    \ ))},
    \ 'workspace_config': { 'intelephense': {
    \   'files.associations': ['*.php'],
    \   'stubs': [
    \        "apache",
    \        "bcmath",
    \        "bz2",
    \        "calendar",
    \        "com_dotnet",
    \        "Core",
    \        "csprng",
    \        "ctype",
    \        "curl",
    \        "date",
    \        "dba",
    \        "dom",
    \        "enchant",
    \        "exif",
    \        "fileinfo",
    \        "filter",
    \        "fpm",
    \        "ftp",
    \        "gd",
    \        "hash",
    \        "iconv",
    \        "imap",
    \        "interbase",
    \        "intl",
    \        "json",
    \        "ldap",
    \        "libxml",
    \        "mbstring",
    \        "mcrypt",
    \        "mssql",
    \        "mysql",
    \        "mysqli",
    \        "oci8",
    \        "odcb",
    \        "openssl",
    \        "password",
    \        "pcntl",
    \        "pcre",
    \        "PDO",
    \        "pdo_ibm",
    \        "pdo_mysql",
    \        "pdo_pgsql",
    \        "pdo_sqlite",
    \        "pgsql",
    \        "Phar",
    \        "posix",
    \        "pspell",
    \        "readline",
    \        "recode",
    \        "Reflection",
    \        "regex",
    \        "session",
    \        "shmop",
    \        "SimpleXML",
    \        "snmp",
    \        "soap",
    \        "sockets",
    \        "sodium",
    \        "SPL",
    \        "sqlite3",
    \        "standard",
    \        "superglobals",
    \        "sybase",
    \        "sysvmsg",
    \        "sysvsem",
    \        "sysvshm",
    \        "tidy",
    \        "tokenizer",
    \        "wddx",
    \        "xml",
    \        "xmlreader",
    \        "xmlrpc",
    \        "xmlwriter",
    \        "Zend OPcache",
    \        "zip",
    \        "zlib",
    \        "wordpress",
    \   ],
    \ }},
    \ })
```

**Comment by bmewburn** [comment]
Glad you got it working. I think this has also come about because of the crash seen in the attached log which may have prevented the default stubs being loaded when config is not available. This is fixed in 1.3.6

**Comment by gbishop** [comment]
Ah, yes, I saw that crash. Thanks for a great tool. I'll be buying a
license.
